xls/public/c_api.h

```diff
@@ -199,6 +213,21 @@ bool xls_bits_eq(const struct xls_bits* a, const struct xls_bits* b);
 // returns 1, `xls_bits_get_bit(bits, 1)` returns 0.
 bool xls_bits_get_bit(const struct xls_bits* bits, int64_t index);
 
+// Returns the bytes in the given bits value. The caller must
+// free the returned `bytes_out` pointer via `free`,  The caller
+// must free the returned `error_out` string via `xls_c_str_free`
+// if an error is returned.
+bool xls_bits_to_bytes(const struct xls_bits* bits, char** error_out,
+                       uint8_t** bytes_out, size_t* byte_count_out);
+
+// Converts the given bits value to a signed or unsigned integer 64-bit integer
+// value. if the conversion is not possible, false is returned and `error_out`
+// contains the error message and the caller must free the `value_out` pointer
+// itself via `free`.
```

Comment:
In the success case, no memory is returned, so there is nothing to free, right?

Reply:
Docs updated.